axiomatic_foundations.rst

```diff
@@ -14,25 +14,21 @@ At the end of this chapter, we will briefly describe another axiomatic framework
 Basic Axioms for Sets
 ---------------------
 
-The axioms of set theory are expressed in first-order logic, for a language with a single binary relation symbol, :math:`\mathord{\in}`. We think of the entire mathematical universe as consisting of nothing but sets; if :math:`x` and :math:`y` are sets, we can express that :math:`x` is an element of :math:`y` by writing :math:`x \in y`. The first axiom says that two sets are equal if and only if they have the same elements:
+The axioms of set theory are expressed in first-order logic, for a language with a single binary relation symbol, :math:`\mathord{\in}`. We think of the entire mathematical universe as consisting of nothing but sets; if :math:`x` and :math:`y` are sets, we can express that :math:`x` is an element of :math:`y` by writing :math:`x \in y`. The first axiom says that two sets are equal if and only if they have the same elements.
```

Comment:
It felt very strange to punctuate the axioms here. Half of them were already preceded by a period instead of a colon, so I switched the others to this style. Does this look okay?